core/tests/protractor_utils/ContributorDashboardPage.js

```diff
@@ -179,6 +182,7 @@ var ContributorDashboardPage = function() {
     await this.waitForOpportunitiesToLoad();
     var opportunity = await _getOpportunityWithHeadingAndSubheading(
       expectedHeading, expectedSubheading);
+    await waitFor(opportunity, 'Opportunity taking too long to appear');
```

Comment:
`waitFor` isn't a function. You would need something like `waitFor.visibilityOf(opportunity, ...)` here. However, we actually already wait for the elements in `_getOpportunityWithHeadingAndSubheading`, so we don't need to wait here at all